web/src/settings_org.ts

```diff
@@ -579,8 +604,6 @@ export function discard_realm_property_element_changes(elem: HTMLElement): void
         case "realm_can_add_custom_emoji_group":
         case "realm_can_access_all_users_group":
         case "realm_can_create_groups":
-        case "realm_can_create_public_channel_group":
-        case "realm_can_create_private_channel_group":
         case "realm_can_create_web_public_channel_group":
```

Comment:
Setting for web-public channels still uses the old UI as we only allow system groups for that setting.

Reply:
OK. This one might deserve some design discussion as to whether there's a good way to achieve our security goals for that one while using the new UI. Options include:
- Deciding to remove the restrictions on role, perhaps replacing it with requiring administrators to be notified about new web-public channels.
- Making the setting just not allow users without at least a moderator role to exercise it (a la how groups work in some other cases), but allow whatever groups to be entered here.

But I'm not sure what's best.